Error running coverage report, traced back to `tao` dependency `core-video-sys`

Hey guys, so I was having some issues compiling sometimes, especially when building my app in order to run `grcov`, my app was having some trouble on macOS, getting the following error message:

```
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-m64" "-L" "/Users/user/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" ... [long linker command here] ...
  = note: Undefined symbols for architecture x86_64:
            "_CFMutableAttributedStringGetTypeID", referenced from:
                _$LT$core_foundation..attributed_string..CFMutableAttributedString$u20$as$u20$core_foundation..base..TCFType$GT$::type_id::h9f6f71bdd347aca0 in foo-cef9f22f855ec251.198s3ihm7i58x886.rcgu.o
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: aborting due to previous error
```

I did some digging and found [this issue](https://github.com/servo/core-foundation-rs/pull/357) - which mentioned that the error was related to an older version of `core-foundation-rs`. 

I did a little more digging and found that `tao` depends on `core-video-sys`, which in turn depends on that older version of `core-foundation-rs`. [That `core-video-sys` crate](https://github.com/LuoZijun/rust-core-video-sys) seems to be abandoned. [Someone graciously cleaned up the code and dependencies](https://github.com/kvark/rust-core-video-sys/tree/cleanup) and submitted a pull request but this has not been merged for nearly 5 months.

While investigating, I was hoping that the PR would fix the issue, and it did in fact. In order to test it I cloned `tao`, and added [a patch statement](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html) to the `Cargo.toml` to use the PR branch. I then added patch statement to `wry` to reference the local `tao` copy with the patch statement in its `Cargo.toml`, and so on and so forth for `tauri-runtime-wry` and `tauri`, and finally my application.

I was wondering how/if the team could address this - perhaps submodule the crate or something? From my understanding crates aren't allowed to be published with git dependencies, but are there any other solutions?

Thanks for all your work, and as always, please let me know how I can help!

--- a/.changes/mac-core-video-sys.md
+++ b/.changes/mac-core-video-sys.md
@@ -1,0 +1,5 @@
+---
+"tao": patch
+---
+
+Fix core-video-sys dependency.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -61,8 +61,8 @@ core-graphics = "0.22"
 dispatch = "0.2"
 scopeguard = "1.1"
 
-  [target."cfg(target_os = \"macos\")".dependencies.core-video-sys]
-  version = "0.1"
+  [target."cfg(target_os = \"macos\")".dependencies.tao-core-video-sys]
+  version = "0.2"
   default_features = false
   features = [ "display_link" ]
 

